doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -188,6 +163,71 @@ preamble similar to the one below, indicating that this is an executable file:
 
 You don't need to edit this preamble, as it is autogenerated.
 
+Links, formatting and images
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When converting a notebook to a MyST Markdown file, you may need to adjust some
```

Comment:
Should we mention at the end of this section that one may use the `rst2myst` utility to convert `.rst` files to MyST Markdown first, before converting them to MyST Markdown notebooks?

Reply:
Honestly, I've never felt the need to use this tool, but I will add a note for completeness.